styles.css

```diff
@@ -1,3 +1,77 @@
+* {
```

Comment:
This is great!! And a good topic to bring up in class. What you're doing here is called "normalization". Each browser by default applies their own basic styles to every html document (For better or worse). Normalizing allows our site to be rendered the same across safari, chrome, firefox, etc.